### PR TITLE
Update pxf cli cluster tool to support PXF on master

### PR DESCRIPTION
**Pull request by ashuka24**
This PR modifies pxf cluster start, stop, restart and status to support the master as well as the standby master host.

### CODE REVIEW

**Review by bradfordb-vmware — APPROVED** [review]

File: cli/go/src/pxf-cli/go.sum [hunk 1]
```diff
@@ -45,8 +45,17 @@ github.com/golang/protobuf v1.2.0/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5y
 github.com/golang/protobuf v1.3.1/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5yJMmIC1U=
 github.com/golang/protobuf v1.3.2 h1:6nsPYzhq5kReh6QImI3k5qWzO4PEbvbIW2cwSfR/6xs=
 github.com/golang/protobuf v1.3.2/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5yJMmIC1U=
+github.com/golang/protobuf v1.4.0-rc.1/go.mod h1:ceaxUfeHdC40wWswd/P6IGgMaK3YpKi5j83Wpe3EHw8=
```
Comment (frankgh) [inline cli/go/src/pxf-cli/go.sum:48]: are these changes required? can we revert this file?

File: cli/go/src/pxf-cli/go.mod [hunk 2]
```diff
@@ -4,20 +4,14 @@ go 1.13
 
 require (
 	github.com/DATA-DOG/go-sqlmock v1.4.1 // indirect
-	github.com/fsnotify/fsnotify v1.4.9 // indirect
 	github.com/gofrs/uuid v3.3.0+incompatible // indirect
 	github.com/greenplum-db/gp-common-go-libs v1.0.4
 	github.com/jackc/pgx v3.6.2+incompatible // indirect
-	github.com/onsi/ginkgo v1.12.0
-	github.com/onsi/gomega v1.9.0
+	github.com/onsi/ginkgo v1.15.0
```
Comment (frankgh) [inline cli/go/src/pxf-cli/go.mod:10]: same here, do we need these changes?

**Review by denalex — APPROVED** [review]
looks good, but we need to look at dependency changes, as others noted

**Review by frankgh — APPROVED** [review]
